charts/delphai-keda/templates/deployment.yml

```diff
@@ -11,7 +11,7 @@ spec:
   selector:
     matchLabels:
       app: {{ .Release.Name }}
-    
+
```

Comment:
:+1: :smile:

Reply:
This should probably be my text editor 😄

Reply:
Yes, and I like this cleanup